l5kit/l5kit/dataset/ego.py

```diff
@@ -76,8 +76,9 @@ def get_frame(self, scene_index: int, state_index: int, track_id: Optional[int]
         """
         frame_interval = self.dataset.scenes[scene_index]["frame_index_interval"]
         frames = self.dataset.frames[frame_interval[0] : frame_interval[1]]
+        agents = self.dataset.agents[frames[0]["agent_index_interval"][0] : frames[-1]["agent_index_interval"][1]]
```

Comment:
This is quite tough to read. Could you do something like:

```suggestion
        start_agent_index = frames[0]["agent_index_interval"][0] 
        end_agent_index = frames[-1]["agent_index_interval"][1]
        agents = self.dataset.agents[start_agent_index:end_agent_index]
```